src/js/api/services/CycleCountApi.js

```diff
@@ -11,13 +11,15 @@ import apiClient from 'utils/apiClient';
 
 export default {
   createRequest: (payload, locationId) => apiClient.post(CYCLE_COUNT_REQUESTS(locationId), payload),
-  startCount: (payload, locationId) => apiClient.post(CYCLE_COUNT_START(locationId), payload),
-  startRecount: (payload, locationId) => apiClient.post(
-    CYCLE_COUNT_RECOUNT_START(locationId), payload,
+  startCount: (payload, locationId, format = null, config = {}) => apiClient.post(
+    CYCLE_COUNT_START(locationId, format), payload, config,
```

Comment:
the arguments list getting huge, we should use it like that in this case:
```suggestion
  startCount: ({ payload, locationId, format = null, config = {} }) => apiClient.post(
    CYCLE_COUNT_START(locationId, format), payload, config,
```